android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -61,15 +61,15 @@ fun SurveyQuestionsScreen(
             questions = questionUiModels
         ),
         onCloseClick,
-        onAnswerClick = viewModel::answerQuestion
+        onAnswerChange = viewModel::answerQuestion
     )
 }
 
 @Composable
 private fun SurveyQuestionsScreenContent(
     uiModel: SurveyQuestionsContentUiModel,
     onCloseClick: () -> Unit = {},
-    onAnswerClick: (Pair<Int, List<Int>>) -> Unit
+    onAnswerChange: (Pair<Int, List<AnswerInput>>) -> Unit
```

Comment:
Should we define the DTO for the whole `Pair<Int, List<AnswerInput>>` data structure?

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/119/commits/a32b80000213f9fff8004060a41930e1bb732233